### PR TITLE
chore(updatecli): provide a manifest to track controller_service_principal_end_date for ci.jenkins.io

### PR DESCRIPTION
**Pull request by smerle33**
as per https://github.com/jenkins-infra/helpdesk/issues/4052

track the end date, it's using the same script as in azure-net.

EDIT:
The short term solution would be to generate a new date directly for the ci.jenkins.io service principal but using this manifest will help us to check if it's working in production as the certificate is due in a week.

The next step will be to migrate the scripts used in the shared-tools as they are the same than in azure-net.

### CODE REVIEW

**Review by dduportal — DISMISSED** [review]
(unrelated to the PR content but: Blocked by https://github.com/jenkins-infra/helpdesk/issues/4046)

**Comment by smerle33** [comment]
> (unrelated to the PR content but: Blocked by [jenkins-infra/helpdesk#4046](https://github.com/jenkins-infra/helpdesk/issues/4046))

should be ok now

File: updatecli/scripts/dateadd.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
-#!/bin/sh
-# This script calculate next date for expiry azurerm_storage_account_sas
+#!/bin/bash
+# This script calculate next date for certificates
```
Comment (dduportal) [inline updatecli/scripts/dateadd.sh:2]:
```suggestion
# This script calculates next date for certificates
```

(nit in a comment: I'm self-addin this)

File: updatecli/scripts/dateadd.sh [hunk 2]
```diff
@@ -1,5 +1,5 @@
-#!/bin/sh
-# This script calculate next date for expiry azurerm_storage_account_sas
```
[the remark this answers is not in the document]
Comment (smerle33) [reply updatecli/scripts/dateadd.sh:2]: yes, we should use the shared-tools for factorization.

[the remark this answers is not in the document]
Comment (dduportal) [reply updatecli/scripts/dateadd.sh:2]: As discussed together: this PR is expected to be a "short term". A set of PRs will be done **after** to factorize the scripts into shared-tools.

**Review by dduportal — COMMENTED** [review]
```
[2024-04-22T11:54:13.658Z] TARGETS
[2024-04-22T11:54:13.658Z] ========
[2024-04-22T11:54:13.658Z] 
[2024-04-22T11:54:13.658Z] updateNextExpiry
[2024-04-22T11:54:13.658Z] ----------------
[2024-04-22T11:54:13.658Z] 
[2024-04-22T11:54:13.658Z] **Dry Run enabled**
[2024-04-22T11:54:13.658Z] 
[2024-04-22T11:54:13.658Z] "ci.jenkins.io.tf" updated with [dry run] content "${1}2024-07-21T00:00:00Z\""
[2024-04-22T11:54:13.658Z] 
[2024-04-22T11:54:13.658Z] ```
[2024-04-22T11:54:13.658Z] --- ci.jenkins.io.tf
[2024-04-22T11:54:13.658Z] +++ ci.jenkins.io.tf
[2024-04-22T11:54:13.658Z] @@ -36,7 +36,7 @@
[2024-04-22T11:54:13.658Z]    controller_service_principal_ids = [
[2024-04-22T11:54:13.659Z]      data.azuread_service_principal.terraform_production.id,
[2024-04-22T11:54:13.659Z]    ]
[2024-04-22T11:54:13.659Z] -  controller_service_principal_end_date = "2024-04-30T00:00:00Z"
[2024-04-22T11:54:13.659Z] +  controller_service_principal_end_date = "2024-07-21T00:00:00Z"
[2024-04-22T11:54:13.659Z]    controller_packer_rg_ids = [
[2024-04-22T11:54:13.659Z]      azurerm_resource_group.packer_images["prod"].id
[2024-04-22T11:54:13.659Z]    ]
[2024-04-22T11:54:13.659Z] 
[2024-04-22T11:54:13.659Z] ```
[2024-04-22T11:54:13.659Z] 
[2024-04-22T11:54:13.659Z] ⚠ - 1 file(s) updated with "${1}2024-07-21T00:00:00Z\"":
[2024-04-22T11:54:13.659Z] 	* ci.jenkins.io.tf
[2024-04-22T11:54:13.659Z] 
[2024-04-22T11:54:13.659Z] 
[2024-04-22T11:54:13.659Z] ACTIONS
[2024-04-22T11:54:13.659Z] ========
[2024-04-22T11:54:13.659Z] 
[2024-04-22T11:54:13.659Z] [Dry Run] An action of kind "github/pullrequest" is expected.
```

LGTM!

**Pull request merged**